### PR TITLE
check-for-stale-keys WIP

### PR DESCRIPTION
**Pull request by calvingiles**
Add logic to notify when ssh keys get more than a year stale.

### CODE REVIEW

File: .github/workflows/check-stale-keys.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
           cd ssh
           git ls-tree -r --name-only HEAD | while read filename; do
            [[ "$(git log -1 --format="%at" -- $filename)" < "$(date  --date='2 years ago' +%s)"  ]] && echo "$(git log -1 --format="%ad" --date=format:'%Y-%m-%dT%H:%M:%S%z' -- $filename) $filename"
-          done || : >> old_keys.txt
+          done >> old_keys.txt || :
```
Comment (JohannDickson) [inline .github/workflows/check-stale-keys.yml:28]: for readability, could also use `/bin/true` to force the return code

Comment (calvingiles) [reply .github/workflows/check-stale-keys.yml]: That is much nicer!

**Review by ekini — APPROVED** [review]
I've added the secret

**Comment by calvingiles** [comment]
> I've added the secret

I reran the workflow, and it doesn’t seem to be triggering the notify I’m slack. I am not sure how to debug as I am not admin in GitHub or slack. Could I get some help or more permissions please? @springload/devops

**Comment by ekini** [comment]
I messed up with the secret name - should be all right now

**Comment by ekini** [comment]
Ah, actually I didn't mess up with the var name, it's just there are 3 different ways to configure it and I used the first on the list.

**Comment by calvingiles** [comment]
I think this is ready to merge once you are happy

**Review by ekini — DISMISSED** [review]

File: .github/workflows/check-stale-keys.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+name: Check stale keys
+on:
+  schedule:
+    - cron: '0 10 * * 1'
+  #  push trigger for testing - remove before merge
```
Comment (JohannDickson) [inline .github/workflows/check-stale-keys.yml:5]: pls remove

**Review by JohannDickson — APPROVED** [review]

**Review by ekini — APPROVED** [review]

**Pull request merged**